### ISSUE TITLE
rmp-serde-0.13.1 beta regression

### ISSUE THREAD

**Issue by est31** [rust-lang/rust#43687]
[rmp-serde](https://crates.io/crates/rmp-serde) version 0.13.1 regressed from stable to beta ([full log](https://cargobomb-reports.s3.amazonaws.com/stable-1.19-beta-1.20.0-beta.1/beta-2017-07-18/gh/euclio.vim-markdown-composer.ba8799cb13f21779ea4b9663ab971de96b3b04ff/log.txt)):

```
Aug 04 09:41:39.372 INFO kablam! error: type `decode::SliceReader<'a>` is private
Aug 04 09:41:39.372 INFO kablam!    --> /cargo-home/registry/src/github.com-1ecc6299db9ec823/rmp-serde-0.13.1/src/lib.rs:285:18
Aug 04 09:41:39.372 INFO kablam!     |
Aug 04 09:41:39.372 INFO kablam! 285 |     let mut de = Deserializer::from_slice(input);
Aug 04 09:41:39.372 INFO kablam!     |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Aug 04 09:41:39.372 INFO kablam! 
```
cc @3Hren

This broke @euclio 's https://github.com/euclio/vim-markdown-composer at commit https://github.com/euclio/vim-markdown-composer/commit/ba8799cb13f21779ea4b9663ab971de96b3b04ff

**Comment by arielb1** [comment]
duplicate of https://github.com/3Hren/msgpack-rust/pull/131, fixed in rmp-serde 0.13.3

**Comment by est31** [comment]
I guess 8e98c7b74ceddb51c981c4889c6b40ee5beab188 fixed this. Thanks @euclio !
